docs/source/user_guide/classifier.rst

```diff
@@ -76,6 +76,7 @@ classifiers: Precision-Recall and ROC.
 
 
 .. ipython:: python
+    :okwarning:
```

Comment:
Why do we need it here?

Reply:
I see it's added to all .rsts

Reply:
Yes, this was the root cause of the initial CI failure after trying to build sphinx docs (for some reason it happened only on python 3.8/ubuntu).

It failed due to this `FutureWarning` on ROC

``` python
        if y_true is not None and y_score is not None:
            warn(
                "ROC will change its signature in version 0.10"
                ", please use ROC.from_raw_data",
                FutureWarning,
                stacklevel=2,
            )
```

By adding `:okwarning:` we can skip this failure so I added it everywhere we used the `roc` function.

I took this warning from [ConfusionMatrix](https://github.com/ploomber/sklearn-evaluation/blob/f495f1cd28ca19367b78ea7d8e82a7ca43166697/src/sklearn_evaluation/plot/classification.py#L62) so I guess it's important since we are also testing it there.

Reply:
Alright